metpy/plots/util.py

```diff
@@ -8,6 +8,28 @@
 from ..units import concatenate
 
 
+def resample_nn_1d(a, centers):
+    """Helper function that returns one-dimensional nearest-neighbor
+    indexes based on user-specified centers.
```

Comment:
Numpy docstring standard says this needs to start with `Parameters`: https://raw.githubusercontent.com/numpy/numpy/master/doc/EXAMPLE_DOCSTRING.rst.txt